readme.md

```diff
@@ -139,7 +177,15 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
+function Instructor (name) {
+  this.name = name;
+  this.receivePresent = function(gift){
+    console.log(this.name + "promptly drops the" + gift + "on the floor");
+  };
+};
+
+var Andy = new Instructor(Andy);
+Andy.receivePresent("Resin Laying Deer Figurine, Gold");
```

Comment:
:+1: 

I would, however, suggest putting all instance methods on the prototype, instead of in the constructor, for efficiency:
```js

function Instructor(name){
  this.name = name;
}
Instructor.prototype.receivePresent = function(gift){
  console.log(this.name + " promptly drops the " + gift + " on the floor.");
}

var andy = new Instructor("Andy");
andy.receivePresent("Resin Laying Deer Figurine, Gold");
```